tools/generators/legacy/src/Generator/ConsolidateTargets.swift

```diff
@@ -272,6 +274,7 @@ extension PlatformAndConfiguration: Comparable {
 struct ConsolidatedTargets: Equatable {
     let keys: [TargetID: ConsolidatedTarget.Key]
     let targets: [ConsolidatedTarget.Key: ConsolidatedTarget]
+    let targetNameMode: TargetNameMode
```

Comment:
I don't think we need this on this type? We can instead pass the value to `Generator.disambiguateTargets()`.

Reply:
Fixed 👍